### PR TITLE
add keda_scaled_object_paused to the docs

### PR DESCRIPTION
**Pull request by geoffrey1330**
related to https://github.com/kedacore/keda/pull/5045

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by geoffrey1330** [comment]
@zroubalik from our conversation here #1111 
Do you mean i should add opentelemetry docs in 2.11?

**Comment by zroubalik** [comment]
> @zroubalik from our conversation here #1111 Do you mean i should add opentelemetry docs in 2.11?

no, add opentelemetry docs and all docs should happen in 2.13, because that's the version we are going to ship the feature in.

**Comment by geoffrey1330** [comment]
> > @zroubalik from our conversation here #1111 Do you mean i should add opentelemetry docs in 2.11?
> 
> no, add opentelemetry docs and all docs should happen in 2.13, because that's the version we are going to ship the feature in.

Alright Let me do that now.

**Comment by geoffrey1330** [comment]
@zroubalik I have made the changes. could you please review.

File: content/docs/2.13/operate/opentelemetry.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The following metrics are being gathered:
 | ------ | ----------- |
 | `keda.build.info` | Info metric, with static information about KEDA build like: version, git commit and Golang runtime info. |
 | `keda.scaler.active` | This metric marks whether the particular scaler is active (value == 1) or in|active (value == 0). |
+| `keda.scaled.object.paused` | This metric indicates whether a scaled object is paused (value == 1) or un|paused (value == 0). |
```
Comment (zroubalik) [inline content/docs/2.13/operate/opentelemetry.md:37]:
```suggestion
| `keda.scaled.object.paused` | This metric indicates whether a ScaledObject is paused (value == 1) or un-paused (value == 0). |
```

File: content/docs/2.13/operate/prometheus.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 
 - `keda_build_info` - Info metric, with static information about KEDA build like: version, git commit and Golang runtime info.
 - `keda_scaler_active` - This metric marks whether the particular scaler is active (value == 1) or in-active (value == 0).
+- `keda_scaled_object_paused` - The paused ScaledObjects in Prometheus metrics (using the annotation detailed [here](https://keda.sh/docs/latest/concepts/scaling-deployments/#pause-autoscaling)).
```
Comment (zroubalik) [inline content/docs/2.13/operate/prometheus.md:15]:
```suggestion
- `keda_scaled_object_paused` - This metric indicates whether a ScaledObject is paused (value == 1) or un|paused (value == 0).
```

File: content/docs/2.13/operate/opentelemetry.md [hunk 3]
```diff
@@ -34,6 +34,7 @@ The following metrics are being gathered:
 | ------ | ----------- |
 | `keda.build.info` | Info metric, with static information about KEDA build like: version, git commit and Golang runtime info. |
 | `keda.scaler.active` | This metric marks whether the particular scaler is active (value == 1) or in|active (value == 0). |
```
Comment (zroubalik) [inline content/docs/2.13/operate/opentelemetry.md:36]:
```suggestion
| `keda.scaler.active` | This metric marks whether the particular scaler is active (value == 1) or in-active (value == 0). |
```

File: content/docs/2.11/operate/prometheus.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_scaler_metrics_latency` - The latency of retrieving current metric from each scaler.
 - `keda_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_scaler_errors_total` - The total number of errors encountered for all scalers.
-- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObject.
+- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObejct.
```
Comment (zroubalik) [inline content/docs/2.11/operate/prometheus.md:19]: why?

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: content/docs/2.11/operate/prometheus.md (hunk 4 above)
Comment (geoffrey1330) [reply content/docs/2.11/operate/prometheus.md:19]: Didn't add the line. i think git decided to treat it as a new change.

Comment (JorTurFer) [reply content/docs/2.11/operate/prometheus.md:19]:
```suggestion
- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObject.
```

**Review by JorTurFer — APPROVED** [review]

File: content/docs/2.13/operate/prometheus.md [hunk 5]
```diff
@@ -12,6 +12,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 
 - `keda_build_info` - Info metric, with static information about KEDA build like: version, git commit and Golang runtime info.
 - `keda_scaler_active` - This metric marks whether the particular scaler is active (value == 1) or in-active (value == 0).
+- `keda_scaled_object_paused` - This metric indicates whether a ScaledObject is paused (value == 1) or un|paused (value == 0).
```
Comment (zroubalik) [inline content/docs/2.13/operate/prometheus.md:15]:
```suggestion
- `keda_scaled_object_paused` - This metric indicates whether a ScaledObject is paused (value == 1) or un-paused (value == 0).
```

**Review by zroubalik — CHANGES_REQUESTED** [review]
typo

**Comment by geoffrey1330** [comment]
@zroubalik I have made the suggested changes. 
cc @JorTurFer

**Review by zroubalik — APPROVED** [review]

**Pull request merged**